diskdict/strtoint.py

```diff
@@ -0,0 +1,123 @@
+import os
+import shutil
+
+import numpy as np
+import cmph
+from diskarray import DiskVarArray
+from deeputil import Dummy
+
+DUMMY_LOG = Dummy()
+
+class VarArray(DiskVarArray):
```

Comment:
@RamanjaneyuluIdavalapati I don't see these being exposed in __init__.py. Please update